### DISCUSSION TITLE
【求助】绝区零二队不知道练什么

### DISCUSSION THREAD

**Discussion by kcl-xgz** [share121/inter-knot#3425] [绝区零]
<img width="275" alt="download" src="https://github.com/user-attachments/assets/2f6641e1-54c4-451f-9e28-9055e664c22b">

绝区零二队不知道练什么角色，有没有大佬帮忙看一下，感谢
![IMG_20240821_221633](https://github.com/user-attachments/assets/13bb8fc6-9e81-494d-80d8-6525c1d8f561)

**Comment by BSNHD** [comment]
如果你没有接着抽限定的想法，就可以把火队练起来；如果还能抽，可以考虑抽简配合格莉丝玩异常紊乱<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Arocohol** [comment]
想抽简嘛，如果抽简就是一队艾莲一队简，如果不抽简就练练玛卡巴卡，11+锤妹/青衣+露西<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >
